AV-45749: Update sample queries (#35)
couchbaselabs/docs-devex: modules/n1ql/pages/n1ql-language-reference/join.adoc

====
Use an ANSI JOIN to list the routes and destination airports that are available from London Heathrow (ICAO code `EGLL`).

> This ANSI JOIN uses the `def_samples_route_sourceairport` index, which can be created by the following statement.

Reviewer: ```suggestion
For this example, use the following statement to create an index called `def_samples_route_sourceairport` index.
```
